### ISSUE TITLE
Only last digit of chktex warning number is shown

### ISSUE THREAD

**Issue by mvcouwen** [latex-lsp/texlab#160]
Only the last digit of the `chktex` warning number is shown in the diagnostics.

For example, I get the warning
```
[chktex 8] [W] Use either `` or '' as an alternative to `"'.
```
while this is in fact warning number 18.

I am using `texlab` through `vim` with `coc.nvim` and `coc-texlab`.

**Comment by pfoerster** [comment]
Thanks for the report. This should be fixed with 3eb23a66541e3660c59728fe8f99c4c85b094cc2.

**Comment by mvcouwen** [comment]
I can confirm that this is fixed.
